src/DecayTreeVectorIntegral.cxx

```diff
@@ -16,6 +16,9 @@ DecayTreeVectorIntegral::DecayTreeVectorIntegral(const DecayTreeVector& dtv)
       Diagonals_(DecayTrees_.size()),
       OffDiagonals_(DecayTrees_.size() - 1)
 {
+    for (size_t i = 0; i < Diagonals_.size(); ++i)
+        Diagonals_[i].value() = 1.;
```

Comment:
The value should be equal to the square of the number of symmetrizations for that `DecayTree`.